### PR TITLE
Meaty ores will now properly drop all kinds of meats/tongues

### PR DESCRIPTION
**Pull request by Incoming5643**
Help I made a dumb mistake and no one called me out on it

### CODE REVIEW

File: code/game/gamemodes/meteor/meteors.dm [hunk 1]
```diff
@@ -256,13 +256,15 @@
...
-		meteordrop += pick(typesof(T))
+	for(var/path in meteordrop)
+		if(path == /obj/item/weapon/reagent_containers/food/snacks/meat/slab/human/mutant)
+			meteordrop -= path
+			meteordrop += pick(subtypesof(path))
+
+	for(var/path in meteordrop)
+		if(path == /obj/item/organ/tongue)
+			meteordrop -= path
+			meteordrop += pick(typesof(path))
```
Comment (coiax) [inline code/game/gamemodes/meteor/meteors.dm]: CHATTER BONE TONGUE

File: code/game/gamemodes/meteor/meteors.dm [hunk 2]
```diff
@@ -256,13 +256,15 @@
...
-
-	for(var/obj/item/organ/tongue/T in meteordrop)
-		meteordrop -= T
-		meteordrop += pick(typesof(T))
+	for(var/path in meteordrop)
+		if(path == /obj/item/weapon/reagent_containers/food/snacks/meat/slab/human/mutant)
+			meteordrop -= path
+			meteordrop += pick(subtypesof(path))
+
+	for(var/path in meteordrop)
```
Comment (RemieRichards) [inline code/game/gamemodes/meteor/meteors.dm]: why loop twice?

Comment (Incoming5643) [inline code/game/gamemodes/meteor/meteors.dm]: One runs on subtypesof, the other typesof, blame /obj/item/weapon/reagent_containers/food/snacks/meat/slab/human/mutant

Comment (coiax) [inline code/game/gamemodes/meteor/meteors.dm]: But everyone knows that /obj/item/weapon/temp/reagent_containers/effect/food/snacks/meat/slab/human/mutant/fly/brown/cooked/weapon/plasteel is the best type path ever!

**Pull request merged**